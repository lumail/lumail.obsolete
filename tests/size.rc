set_selected_folder('output/folders/size')
io.write(('Messages: %d\n'):format(count_messages()))
local idx = 0
while idx < count_messages() do
    jump_index_to(idx)
    local msg = current_message()
    io.write('size: '..msg:size()..'\n')
    idx = idx + 1
end
