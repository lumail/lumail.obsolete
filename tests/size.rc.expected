Messages: 1
size: 150
Exit: 0
